build.gradle

```diff
@@ -1,30 +1,21 @@
-group 'uk.gov.verify'
-version '1.0-SNAPSHOT'
-
-
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.github.jengelman.gradle.plugins:shadow:5.0.0'
-    }
+plugins {
+    id 'com.github.johnrengelman.shadow' version '5.2.0'
 }
 
-apply plugin: 'com.github.johnrengelman.shadow'
 apply plugin: 'java'
 
+group 'uk.gov.verify'
+version '1.0-SNAPSHOT'
+
 def dependencyVersions = [
         dropwizard:"1.3.5",
-        event_emmiter: "0.0.1-58"
+        event_emmiter: "2.0.0-82"
```

Comment:
```suggestion
        event_emmiter: "2.0.0-83"
```

Reply:
Excellent spot, thanks Lewis! Have updated to `83`